tests/functional/generate_pages.py

```diff
@@ -257,11 +257,9 @@ def generate_pascal_case_from_id(id_str):
     return name
```

Comment:
How come this file changed in this PR?

Reply:
Because I removed the debug line after discussing with @ajmaddaford - the other changes are small improvements